### PR TITLE
Prepare marking getGaussNodesAndWeights private

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: lib/src/Uncertainty/Algorithm/Analytical/AnalyticalResult.cxx [hunk 1]
```diff
@@ -233,15 +234,16 @@ void AnalyticalResult::computeMeanPointInStandardEventDomain() const
   Scalar scaling = hasoferReliabilityIndex_ * p_standardDistribution->computeRadialDistributionCDF(hasoferReliabilityIndex_, true);
   Scalar a = hasoferReliabilityIndex_;
   // Quadrature rule
-  Point weights;
-  const Point nodes(p_standardDistribution->getGaussNodesAndWeights(weights));
-  const UnsignedInteger nodesSize = nodes.getSize();
+  const UnsignedInteger integrationNodesNumber = ResourceMap::GetAsUnsignedInteger("Distribution-DefaultIntegrationNodesNumber");
```
Comment (sofianehaddad) [inline lib/src/Uncertainty/Algorithm/Analytical/AnalyticalResult.cxx]: Maybe we should also rename the RM key? Distribution-DefaultIntegrationNodesNumber should be removed also

Comment (regislebrun) [inline lib/src/Uncertainty/Algorithm/Analytical/AnalyticalResult.cxx]: @sofianehaddad The key Distribution-DefaultIntegrationNodesNumber is used to tune the internal integrations done in many distributions. Don't touch it!

Comment (jschueller) [inline lib/src/Uncertainty/Algorithm/Analytical/AnalyticalResult.cxx]: The thing is getGaussNodesAndWeights (and integrationNodesNumber_...) will continue to be used internally as the number of nodes has to be changed depending on the distribution through setIntegrationNodesNumber (see Student, MaxEntropy, RandomMixture, Conditional, Normal...)

Comment (sofianehaddad) [inline lib/src/Uncertainty/Algorithm/Analytical/AnalyticalResult.cxx]: @jschueller for these distributions, we use other keys (Student-MarginalIntegrationNodesNumber",  "MaximumEntropyOrderStatisticsDistribution-CDFIntegrationNodesNumber", ...)

@regislebrun ok for internal use but the specific AnalyticalResult should have it is own key

**Comment by regislebrun** [comment]
+1

File: lib/src/Uncertainty/Algorithm/Analytical/AnalyticalResult.cxx (hunk 1 above)
Comment (jschueller) [inline lib/src/Uncertainty/Algorithm/Analytical/AnalyticalResult.cxx]: ok, done

**Pull request merged**